src/sql/platform/accounts/common/accountStore.ts

```diff
@@ -47,6 +47,15 @@ export default class AccountStore implements IAccountStore {
 		});
 	}
 
+	public removeAllAccounts(): Thenable<boolean> {
+		return this.doOperation(() => {
+			return this.readFromMemento()
+				.then(() => this.removeAllAccountsList())
+				.then(result => this.writeToMemento(result.updatedAccounts).then(() => result))
+				.then(result => result.accountRemoved);
```

Comment:
also, why do we need an extra step to return the result.accountRemoved?

Reply:
Just following how this file has been written before this.